packages/dialtone-vue2/recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -116,7 +116,6 @@ export default {
       return this.i18n.$t('DIALTONE_EMOJI_ROW_REACTION_LABEL', {
         reaction: getEmojiShortCode(reaction.emojiUnicodeOrShortname),
         personCount: reaction.num,
-        youIncluded: reaction.isSelected,
```

Comment:
don't we still need this logic in some other way even if fluent doesn't support booleans? Wouldn't it still need to output the word "you" which needs to be translated?